src/collection.js

```diff
@@ -407,8 +407,12 @@ export default class Collection {
     if (!this.idSchema.validate(record.id)) {
       return Promise.reject(new Error(`Invalid Id: ${record.id}`));
     }
+
     return this.get(record.id)
-      .then(_ => {
+      .then(res => {
+        if (res.data.last_modified && !record.last_modified) {
```

Comment:
Good catch. I wouldn't think of this case but know that you talk about it, it seems totally a good catch :+1: